gem/spec/fixtures/carthage_after/Example.xcodeproj/project.pbxproj

```diff
@@ -7,10 +7,11 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
-		5B5678F4326B273D6B2DD1F4 /* EarlGrey.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 0B657D1D6749CAC16B3688D5 /* EarlGrey.framework */; };
+		7D3DE2B2D4B6ADBAD8023E1F /* EarlGrey.swift in Sources */ = {isa = PBXBuildFile; fileRef = 23083A8BBED0E8BDFEA08B61 /* EarlGrey.swift */; };
+		9F0F538927F2B2C5677F9D15 /* EarlGrey.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = E96DE017C569DE30A62E26B5 /* EarlGrey.framework */; };
```

Comment:
One is for frameworks and the other is for the copy files script. Cocoapods is setup the same way if you look at that fixture.

```ruby
user_project.products_group.new_file(framework_path)

test_target.frameworks_build_phase.add_file_reference framework_ref
```

are you saying it'd be better to cache the framework ref and use the same one in both locations?